(DOCSP-11402): Remove repetition in Don't Humanize Inanimate Objects (#66)
mongodb/docs-meta: source/style-guide/writing/use-effective-verbs.txt

inanimate objects. For example, a web application doesn't *know*,
*need*, *remember*, *see*, *think*, *understand*, or *want*.

> Often, we use anthropomorphic words in an attempt to make difficult topics more understandable or relatable. This can actually make the

Reviewer: **suggestion:** As written, this is hard to follow. I would recommend condensing.

```suggestion
Often, we use words that anthropomorphize (or humanize) to simplify topics. This can actually make the
```

Author: Reworded.

inanimate objects. For example, a web application doesn't *know*,
*need*, *remember*, *see*, *think*, *understand*, or *want*.

> Often, we use anthropomorphic words to make difficult topics more

Reviewer: **suggestion:** The use of *anthropomorphic* was why I added "humanize" first so people could understand that the word means the same. Consider adding that back in.

Also, do they have to be relatable or understandable? Could they be one or the other?

> Often, we use anthropomorphic words to make difficult topics more
> understandable or relatable. These words can actually make the
> concept harder to grasp for:

Reviewer: ```suggestion
Often, we humanize technology with anthropomorphic words.
Some writers believe this makes difficult topics more
understandable or relatable. These words can make the
concept harder to grasp for:
```
